src/components/combinedgraphs.tsx

```diff
@@ -86,6 +86,9 @@ export const SelectFilters = (props: FilterEditorProps): null | JSX.Element => {
     if (query.context) {
       delete query.context[filtername];
     }
+    if (query.params && query.params.selections && query.params.selections.context) {
```

Comment:
This looks like an opportunity to use the ".?" operator.

Reply:
This is embarrassing…